posts/2020-02-26-rustconf-cfp.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: "The 2020 RustConf CFP is Now Open!"
+author: Rust Community
+description: "The call for proposals for RustConf 202 is open; We want to hear from you!"
+---
+
+Greetings fellow Rustaceans!
+
+The 2020 RustConf Call for Proposals is now open!
+
+Got something to share about Rust? Want to talk about the experience of learning and using Rust? Want to dive deep into an aspect of the language? Got something different in mind? We want to hear from you! The [RustConf 2020 CFP site](https://cfp.rustconf.com/events/rustconf-2020) is now up and accepting proposals.
+
+If you may be interested in speaking but aren't quite ready to submit a proposal yet, we are here to help you. We will be holding speaker office hours regularly throughout the proposal process, after the proposal process, and up to RustConf itself on August 20 and 21, 2020. We are available to brainstorm ideas for proposals, talk through proposals, and provide support throughout the entire speaking journey. We need a variety of perspectives, interests, and experience levels for RustConf to be the best that it can be - if you have questions or want to talk through things please don't hesitate to reach out to us! Watch this blog for more details on speaker office hours - they will be posted very soon.
+
+The RustConf CFP will be open through Monday, April 5, 2020, hope to see your proposal soon!
```

Comment:
This might a US english thing but I think this is correct.
```suggestion
The RustConf CFP will be open through Monday, April 5th, 2020, hope to see your proposal soon!
```